composer.json

```
@@ -52,13 +52,13 @@
         "prestashop/ps_banner": "^2",
         "prestashop/ps_categorytree": "^2",
         "prestashop/ps_checkpayment": "^2",
-        "prestashop/ps_contactinfo": "^3",
+        "prestashop/ps_contactinfo": "^3.2",
         "prestashop/ps_currencyselector": "^2",
         "prestashop/ps_customeraccountlinks": "^3",
         "prestashop/ps_customersignin": "^2",
         "prestashop/ps_customtext": "^4",
-        "prestashop/ps_emailsubscription": "^2",
-        "prestashop/ps_facetedsearch": "^3",
+        "prestashop/ps_emailsubscription": "^2.5",
+        "prestashop/ps_facetedsearch": "^3.2",
```

```suggestion
        "prestashop/ps_facetedsearch": "^3.2.1",
```
The 3.2.0 introduces many bugs :)

---

:ok_hand: